### ISSUE TITLE
Reorganize the graphs on the analyze page

### ISSUE THREAD

**Issue by SeanKennyNF** [PathoTracker/Pathotracker#144]
From Harriet [here](https://3.basecamp.com/5061858/buckets/30589330/messages/6958342546#__recording_6962615947),

> Graph positions:
> Based on Heidelberg feedback, let's reposition the graphs as follows
> 1. Estimate count by WHO region and arbovirus (this should also be the one featured on the dashboard page for now)
> 2. Median seroprevalence of arboviruses by WHO Region
> 3. Change in median seroprevalence over time
> 4. Estimate count by arbovirus & antibody type
> 5. Cumulative estimate count over time by sample frame
> 6. Median seroprevalence by WHO region and age group
> **Remove Cumulative estimate count over time by arbovirus and Top ten countries reporting estimates by arbovirus

Let's change the order of the graphs on the analyze page to match this order and remove the graphs she wants removed.

**Comment by harrietware** [comment]
Hi team! We're planning to conduct more user interviews the week of Feb 5-9 – I'd love to see this, #145, #146, and #148 implemented by then so we can get new feedback on the visualizations :)

**Comment by SeanKennyNF** [comment]
Hey Harriet, sorry it took me a while to get to this, crazy week. I can take care of https://github.com/PathoTracker/Pathotracker/issues/145 and https://github.com/PathoTracker/Pathotracker/issues/146 today and get them shipped tomorrow no problem. As far as https://github.com/PathoTracker/Pathotracker/issues/108 is concerned, @EliasPinno was looking at that last so I'll see what we can do there. We're meeting on Sunday so we'll talk then.

Long story short, I can definitely promise https://github.com/PathoTracker/Pathotracker/issues/145 and https://github.com/PathoTracker/Pathotracker/issues/146 by Monday and I'll give https://github.com/PathoTracker/Pathotracker/issues/108 a maybe if that's okay :)

**Comment by SeanKennyNF** [comment]
I'll give an update here Monday morning

**Comment by SeanKennyNF** [comment]
Oops sorry @harrietware. I realized I forgot to tag you on my previous message.

#144 up for review here (https://github.com/PathoTracker/Pathotracker/pull/154) and most likely good for Monday.
#145 up for review here (https://github.com/PathoTracker/Pathotracker/pull/155) and most likely good for Monday.
#146 up for review here (https://github.com/PathoTracker/Pathotracker/pull/156) and most likely good for Monday.
#108 - I'll talk to Elias tomorrow and hopefully we can make some progress then.

**Comment by SeanKennyNF** [comment]
Another update here - https://github.com/PathoTracker/Pathotracker/issues/108#issuecomment-1925590404
